apps/settings/test/unit/_developer.html

```diff
@@ -0,0 +1,294 @@
+<gaia-header action="back" data-href="#root">
```

Comment:
I think we can only import the part that is used in the tests instead of the entire html.